dronekit/__init__.py

```diff
@@ -110,6 +111,7 @@ def __init__(self, master, status_printer=None):
         self.epv = None
         self.satellites_visible = None
         self.fix_type = None  # FIXME support multiple GPSs per vehicle - possibly by using componentId
+        self.ekf_ok = False
```

Comment:
Opened another issue for that. (will want to keep ekf_ok for the betas)

https://github.com/dronekit/dronekit-python/issues/336